app/lib/question_navigation.rb

```diff
@@ -163,4 +163,30 @@ class QuestionNavigation
     Questions::SuccessfullySubmittedController,
     Questions::FeedbackController,
   ].freeze
+
+  def self.determine_current_question(intake)
```

Comment:
I refactored this into a background job, but the background jobs fails because we're grabbing data from controllers that aren't available/set up correctly when running outside of the application flow. There's a larger refactor that can be done to deal with that issue and not rely on the controllers at all, but all of the current "Flow" logic is based on controllers. This doesn't seem like the right time to do that refactor, and I don't want to spend too much more time on the code path for backfilling -- this should work fine, and I've added TODOs to remove at the end of the season.